src/components/Attachments/Attachment.js

```diff
@@ -24,9 +24,7 @@ const Attachment = ({ attachment, host }) => {
       <p>{attachment?.description}</p>
       <p>
         <Icon name="file" size="x20" />
-        <a href={`http://localhost:3000${attachment.title_link}`}>
-          {attachment.title}
-        </a>
+        <a href={`${host}${attachment.title_link}`}>{attachment.title}</a>
```

Comment:
I think you've already added this change?

Reply:
Yes

Reply:
Do we need to add it again?

Reply:
No.

Reply:
Hey btw I was working on input-toolbar visibilty, is this good to have?

[Screencast from 18-02-23 04:53:40 PM IST.webm](https://user-images.githubusercontent.com/92238941/219860087-a8239444-588e-4802-b24f-0a8a55572d3f.webm)